src/mesido/workflows/grow_workflow.py

```diff
@@ -182,17 +182,23 @@ def read(self):
         super().read()
 
         # Error checking:
-        # - installed capacity/power of a heating demand is sufficient for the specified heat
+        # - installed capacity/power of a heating/cooling demand is sufficient for the specified
         #   demand profile
-        for key, vals in self._asset_potential_errors.items():
-            if "heat_demand.power" in key:
-                for val in vals:
-                    print(val)
-                logger.error(
-                    "Heating demand insufficient installed capacity: please increase the installed"
-                    " power or reduce the demand profile peak value of the demands listed."
-                )
-                exit(1)
+        is_error = False
+        for error_type, errors in self._asset_potential_errors.items():
+            if error_type in ["heat_demand.power", "cold_demand.power"]:
+                if len(errors) > 0:
+                    for asset_name in errors:
+                        print(self._asset_potential_errors[error_type][asset_name])
```

Comment:
printing will not necessary log the error.

Reply:
print has been updated to logger.error